apps/web/middleware.ts

```diff
@@ -1,8 +1,25 @@
 import { collectEvents } from "next-collect/server";
+// eslint-disable-next-line @next/next/no-server-import-in-page
+import { NextMiddleware, NextResponse } from "next/server";
 
 import { extendEventData, nextCollectBasicSettings } from "@calcom/lib/telemetry";
 
+const V2_WHITELIST = ["/settings/admin"];
```

Comment:
I will possibly explore a cleaner way to do this - wonder if we can somehow glob the V2 directory ?

Reply:
Don't hive it much of a thought. It will all be gone after V2 launch anyways @sean-brydon